simtools/applications/db_development_tools/add_model_parameters_from_repository_to_db.py

```diff
@@ -109,38 +120,45 @@ def add_values_from_json_to_db(file, collection, db, db_name, file_prefix, logge
     )
 
 
-def _add_metadata_to_db(db, db_name, logger):
+def _add_metadata_to_db(args_dict, db, logger):
     """
     Add metadata to the DB.
 
     Parameters
     ----------
-    db : DatabaseHandler
-        Database handler object.
+    args_dict : dict
+        Command line arguments.
     db_name : str
         Name of the database to be created.
     logger : logging.Logger
         Logger object.
     """
     logger.info("Adding metadata to the DB")
+    input_path = Path(args_dict["input_path"])
+    simulation_model_tags = gen.collect_data_from_file_or_dict(
+        file_name=input_path / "simulation_model_tags.json", in_dict=None
```

Comment:
Adding a new model version means to add a new directory to the gitlab etc (documentation updates are currently in preparation). So adding an entry into the model version json file is one step of this workflow.

Note that this is only necessary because we want to keep the possibility to call productions 'Prod6', 'Prod5' plus have the actually versioning with a date (this is what we have decided a while ago). If this wouldn't be necessary, one could simply read the directory names in the model_parameters repository. Anyway, I would keep this for now and see if we simplify in future.

Schema file: yes, and no - this is a very simple structure and I think the logic is absolutely clear from the examples. So probably no need at this point for a schema file.

Reply:
Also note that this part is database maintenance and not foreseen to be done by the 'normal' user.

Reply:
"The logic is absolutely clear" is not a very stable or convincing argument. Simtools does not have any idea at the moment on how this file should look like. This is for example different for the model parameter files (in the model parameters repo), because simtools have the schema files and know what to expect. So either we put that file in simtools or we create a schema file for it in simtools.

Reply:
The schema file would only allow to validate the format of the yaml file. As this is a single field dict, I am not sure if we need to introduce a schema.